### PR TITLE
Fix inverted transforms

### PR DESCRIPTION
**Pull request by emersion**
This patch makes it so users that have configured their screen with a
transform don't have to update their config after the wlroots breaking
change.

References: https://github.com/swaywm/wlroots/pull/2023

### CODE REVIEW

File: sway/commands/output/transform.c [hunk 1]
```diff
@@ -32,6 +33,11 @@ struct cmd_results *output_cmd_transform(int argc, char **argv) {
 	} else {
 		return cmd_results_new(CMD_INVALID, "Invalid output transform.");
 	}
+
+	// Sway uses clockwise transforms, while WL_OUTPUT_TRANSFORM_* describe
+	// anti-clockwise transforms
+	transform = wlr_output_transform_invert(transform);
```
Comment (mstoeckl) [inline sway/commands/output/transform.c]: While `wlr_output_transform_invert` correctly fixes the rotation direction for pure rotations, the inverse transformation of `WL_OUTPUT_TRANSFORM_FLIPPED_90` is itself, not the value `WL_OUTPUT_TRANSFORM_FLIPPED_270` that the wlroots patch swapped it with.

Comment (emersion) [reply sway/commands/output/transform.c]: Oh, right. I guess the clockwise/anticlockwise handling needs to be fixed as well.

**Comment by emersion** [comment]
Updated, also added a note in the docs.

**Comment by ddevault** [comment]
Thanks!

**Pull request merged**